src/library/export/coverartcopyworker.cpp

```diff
@@ -0,0 +1,113 @@
+#include "library/export/coverartcopyworker.h"
+
+#include <QDebug>
+#include <QFileInfo>
+#include <QMessageBox>
+
+#include "util/imagefiledata.h"
+#include "util/safelywritablefile.h"
+
+namespace {
+
+const bool kUseTemporaryCoverArt = true;
+
+const bool kUseTemporaryFileWithPrefix = false;
+
+} // namespace
+
+void CoverArtCopyWorker::run() {
+    m_isCoverArtUpdated = false;
+    copyFile(m_coverArtImage, m_coverArtAbsolutePath);
+}
+
+void CoverArtCopyWorker::copyFile(
+        const ImageFileData& m_coverArtImage,
+        const QString& m_coverArtAbsolutePath) {
+    QFileInfo coverArtPathFileInfo(m_coverArtAbsolutePath);
+
+    if (coverArtPathFileInfo.exists()) {
+        switch (makeOverwriteRequest(m_coverArtAbsolutePath)) {
+        case OverwriteAnswer::CANCEL:
+            return;
+        case OverwriteAnswer::OVERWRITE:
+            break;
+        }
+
+        mixxx::SafelyWritableFile safelyWritableFile(m_coverArtAbsolutePath,
+                kUseTemporaryCoverArt,
```

Comment:
We don't want the copy right?

Reply:
Copy is not used with the latest changes 👍